opal/mca/threads/base/mutex.c

```diff
@@ -25,13 +25,8 @@
  */
```

Comment:
Not to be picky, but the current git history indicates that our pthread support was derived from Argobots. I would definitively prefer to have the facts.

Reply:
I think it's a GitHub problem.
```
$ git log:
commit f434d5c6e8c6112ad90266ee497712c14c315f5b (HEAD -> topic/thread_remove_dup)
Author: Shintaro Iwasaki <siwasaki@anl.gov>
Date:   Thu Apr 8 17:31:06 2021 -0500

    opal/mca/threads: move mutex implementation to threads/base
    
    Now implementations of opal_mutex_xxx() and opal_cond_xxx() are the
    same.  This patch moves them to base/mutex.c and mutex.h and removes the
    duplication.
    
    Note that those implementations are copied from
    pthread/threads_pthreads_mutex.c and pthread/threads_pthreads_mutex.h.
    
    Signed-off-by: Shintaro Iwasaki <siwasaki@anl.gov>
...
$ git diff --stat HEAD^
 ...
 opal/mca/threads/base/Makefile.am                                    |   2 +
 opal/mca/threads/{pthreads/threads_pthreads_mutex.c => base/mutex.c} |   1 -
 ...
```
I explicitly used `git mv` to move this Pthreads code, so the git history should be correct. As I cannot control how GitHub shows (maybe there's a way to do such), I'd really appreciate it if you'd be less picky for this.

Reply:
If it's a github issue then we're all good.